Assignment_1.py

```diff
@@ -0,0 +1,180 @@
+# %%
+from statistics import stdev
+import pandas as pd
+import numpy as np
+
+from nilearn import plotting
+import nibabel as nib
+
+from obspy.signal.detrend import polynomial
+from statsmodels.tsa.stattools import adfuller
+from scipy import signal
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+import warnings
+# %%
+
+
+def detrend_polynomial(dta):
```

Comment:
Only a simple linear detrending is asked for in the assignment. It's a one liner with [`scipy.signal.detrend`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.detrend.html) if you keep the data as a numpy array. Once you start packing it as a pandas `DataFrame` you are making things difficult for yourself.

Reply:
Good to know. I will try the linear detrend. That makes my work much easier 😄😄.